### PR TITLE
INTEGR-164: add new connector City24

### PR DESCRIPTION
**Pull request by SergeKoTnk**

### CODE REVIEW

File: data/vendors.json [hunk 1]
```diff
@@ -754,6 +754,15 @@
       "en": "Perfect Money"
     }
   },
+  {
+    "code": "city24",
+    "status": "active",
+    "name": {
+      "en": "CITY24",
```
Comment (TonyTramb) [inline data/vendors.json:761]: image

File: data/vendors.json [hunk 2]
```diff
@@ -754,6 +754,15 @@
       "en": "Perfect Money"
     }
   },
+  {
+    "code": "city24",
+    "status": "active",
+    "name": {
```
Comment (TonyTramb) [inline data/vendors.json:760]: site: https://city24.ua/

File: data/payment_providers.json [hunk 3]
```diff
@@ -12933,6 +12933,19 @@
...
   },
+  {
+    "status": "active",
+    "code": "city24",
+    "vendor": "city24",
+    "payout_method": [
+      "payment_card"
+    ],
+    "name": {
+      "en": "CITY24",
```
Comment (TonyTramb) [inline data/payment_providers.json:12944]: City24

Comment (SergeKoTnk) [reply data/payment_providers.json:12944]: done

File: data/vendors.json (hunk 2 above)
Comment (SergeKoTnk) [reply data/vendors.json:760]: added

File: data/vendors.json (hunk 1 above)
Comment (SergeKoTnk) [reply data/vendors.json:761]: done

**Review by TonyTramb — APPROVED** [review]

**Pull request merged**